plugins/include-media-export/includeMedia.js

```diff
@@ -0,0 +1,82 @@
+var im = (function () {
+  var element = document.body;
+  var autoUpdate = true;
+  var breakpoints = false;
+
+  function setElement(newElement) {
+    element = newElement;
+  }
+
+  function setUpdateMode(updateMode) {
+    if (updateMode == 'manual') {
+      autoUpdate = false;
+      readBreakpoints();
+    }
+  }
+
+  function readBreakpoints() {
+    if (window.getComputedStyle && (window.getComputedStyle(element, '::after').content != '')) {
+      var data = window.getComputedStyle(element, '::after').content.replace(/'/g, '');
+
+      try {
+        breakpoints = JSON.parse(data);
+      } catch(err) {}
+    }
+  }
+
+  function isBreakpointActive(breakpoint) {
+    if (autoUpdate) {
+      readBreakpoints();
+    }
+
+    return breakpoints.hasOwnProperty(breakpoint) &&
+           breakpoints[breakpoint].hasOwnProperty('active') && 
+           breakpoints[breakpoint].active;
+  }
+
+  function isBreakpointNotActive(breakpoint) {
+    return !isBreakpointActive(breakpoint);
+  }
+
+  function getActiveBreakpoint() {
+    if (autoUpdate) {
+      readBreakpoints();
+    }
+
+    var largest = {name: false, value: 0};
+
+    for (var breakpoint in breakpoints) {
+      if (breakpoints.hasOwnProperty(breakpoint) && breakpoints[breakpoint].hasOwnProperty('active')) {
+        var breakpointValue = parseFloat(breakpoints[breakpoint].value);
+
+        if (breakpoints[breakpoint].active && (breakpointValue > largest.value)) {
+          largest = {name: breakpoint, value: breakpointValue};
+        }
+      }
+    }
+
+    return largest.name;
+  }
+
+  function getBreakpointValue(breakpoint, asNumber) {
+    if (autoUpdate) {
+      readBreakpoints();
+    }
+
+    if (!breakpoints || !breakpoints.hasOwnProperty(breakpoint)) {
+      return false;
+    }
+
+    return asNumber ? parseFloat(breakpoints[breakpoint].value) : breakpoints[breakpoint].value;
```

Comment:
Idea: instead of `asNumber` relying on `parseFloat`, accept a function that is being applied on the value.

```javascript
if (typeof fn === 'function') {
  return fn(breakpoints[breakpoint].value);
}
```

```javascript
var value = im.getBreakpointValue('small', parseFloat);
```